android/design-system/src/main/java/com/adammcneilly/pocketleague/android/designsystem/teamselection/TeamSelectionListItem.kt

```diff
@@ -47,9 +51,31 @@ fun TeamSelectionListItem(
     Row(
         modifier = modifier
             .fillMaxWidth()
-            .padding(16.dp),
-        verticalAlignment = Alignment.CenterVertically,
+            .padding(25.dp),
+        horizontalArrangement = Arrangement.SpaceBetween
```

Comment:
Thanks for this change! The image is helpful, but we should revert a couple changes to the row setup here. 

1. Keeping the padding at 16.dp is consistent with spacing and list items elsewhere in the app. Ideally I remove these hardcodes and put it as part of the theme. 
2. We want to keep the verticalAlignment property as `CenterVertically` because this keeps the text centered vertically which is especially helpful given the size of the image, too. 

If we can update these changes I'm happy to merge this. 

```suggestion
            .padding(16.dp),
        verticalAlignment = Alignment.CenterVertically,
        horizontalArrangement = Arrangement.SpaceBetween,
```

Reply:
Also I've triggered the CI builds so you should be able to see those logs and find out if any tests break as a result (unlikely, but there may be lint checks that fail).

Reply:
ok, checking this,,,failed due to a wildcard import, can compromise this rule later maybe? [😂](https://emojipedia.org/face-with-tears-of-joy/)

Reply:
Hmm does running `./gradlew formatKotlin` auto fix it by chance?

Reply:
yeap,this finally worked,pushing changes in a few

Reply:
Great, thanks! I'll approve once the next run of checks passes.